Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -14,34 +14,68 @@ UCLASS()
 class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 {
 	GENERATED_BODY()
-public:
-	ABenchmarkGymGameModeBase();
 
-protected:
+	class FActorCountInfo
+	{
+	public:
 
-	static FString ReadFromCommandLineKey;
+		explicit FActorCountInfo(int32 InTotal)
+			: Total(InTotal)
+			, SmoothedTotal(InTotal)
+		{}
+
+		explicit FActorCountInfo()
+			: Total(0)
+			, SmoothedTotal(0.0f)
+		{}
+
+		void SetTotal(int32 InTotal)
+		{
+			Total = InTotal;
+			SmoothedTotal = SmoothedTotal * 0.9f + Total * 0.1f;
+		}
 
-	struct FExpectedActorCount
+		int32 GetTotal() const { return Total; }
+		int32 GetSmoothedTotal() const { return FMath::RoundToInt(SmoothedTotal); }
```

Comment:
Nit: Int truncation here might neglect 0.5 around the min and allow 0.5 more around max 
```suggestion
		int32 GetSmoothedTotal() const { return FMath::RoundToInt(SmoothedTotal + 0.5f); }
```

Reply:
Its a rounding function it does not truncate? 
`	static FORCEINLINE int32 RoundToInt(float F)
	{
		return FloorToInt(F + 0.5f);
	}`

Reply:
FMath::RoundToInt will round 0.5 up. Perhaps I miss understand?